src/QsCompiler/Compiler/CompilationLoader.cs

```diff
@@ -283,10 +294,18 @@ public CompilationLoader(SourceLoader loadSources, ReferenceLoader loadReference
             {
                 this.CompilationStatus.Monomorphization = 0;
                 void onException(Exception ex) => this.LogAndUpdate(ref this.CompilationStatus.Monomorphization, ex); 
-                var succeeded = this.CompilationOutput != null && this.CompilationOutput.Monomorphisize(out this.CompilationOutput, onException);
+                bool succeeded = this.CompilationOutput != null && this.CompilationOutput.Monomorphisize(out this.CompilationOutput, onException);
```

Comment:
I specifically changed this from a var to a bool. I am not fond of overuse of the var keyword in C# because it contributes to a style of coding that is hard to read. This particular case wasn't too bad to understand, but to the same effect it could never _not_ be a bool, so choosing to change the data structure somewhere would not change the type here.